src/services/dust_bin_service.ts

```diff
@@ -0,0 +1,63 @@
+import { Service } from "homebridge";
+import { BehaviorSubject, distinct, filter } from "rxjs";
+import { CoreContext } from "./types";
+import { callbackify } from "../utils/callbackify";
+import { PluginServiceClass } from "./plugin_service_class";
+import { CharacteristicValue } from "hap-nodejs/dist/types";
+
+export interface DustBinConfig {
+  dustBin: boolean;
+}
+
+const DUST_BIN_FULL_ERROR_CODES = ["10"];
```

Comment:
@luc-ass for now, the plugin will only react to the error code "10". Let me know if that is enough of if it should react to another code instead.

Reply:
I'll keep an eye out. What is the reaction supposed to look like?